packages/cli/src/command.js

```diff
@@ -35,6 +36,12 @@ export default class NuxtCommand extends Hookable {
   }
 
   async run () {
+    await this.callHook('setup', {
+      argv: this._argv,
+      cmd: this.cmd,
+      rootDir: path.resolve(this.argv._[0] || '.')
```

Comment:
That's a duplicated logic here but at least more consistent to be into the core :)